packages/api/src/external/commonwell/patient-external-data.ts

```diff
@@ -97,3 +98,56 @@ export const setCommonwellId = async ({
     return updatedPatient.update({ data: updatedData }, { transaction });
   });
 };
+
+export async function updatePatientScheduledQueryRequestId({
```

Comment:
Another function to update a patient's field... 😅  Created a ticket to fix this: https://github.com/metriport/metriport-internal/issues/1646

Can we rename from "update" to "reset"? Update indicates we're sending the updated data, reset is clearer that we're resetting it. And add "Doc" so its clearer what's doing?